src/Service/BoardService.php

```diff
@@ -0,0 +1,48 @@
+<?php
+    namespace App\Service;
+
+    use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+    class BoardService
+    {
+        public const BOARD_LENGTH = 6;
+
+        public function initBoard(): array
+        {
+            // Return a 6*6 2 dimensions array filled with 0 as value for each cell.
+            return array_fill(0, 6, array_fill(0, 6, 0));
+        }
+
+
+        private function isOutsideBoard($position): bool {
+            if ($position[0] < 0 || $position[0] >= 6 || $position[1] < 0 || $position[1] >= 6) {
+                return true;
+            }
+
+            return false;
+        }
+
+        public function isPositionAvailable($board, $position): bool
+        {
+            if($this->isOutsideBoard($position) == true) {
```

Comment:
no need to add `==true`. PHP is different between one or two egal (1 == true is true but 1 === true is false